api/api_ethereum.go

```diff
@@ -773,7 +791,18 @@ func newEthRPCTransaction(tx *types.Transaction, blockHash common.Hash, blockNum
 		result.TransactionIndex = (*hexutil.Uint64)(&index)
 	}
 
-	// TODO-Klaytn: Have to add additional fields for ethereum transaction types.
+	switch typeInt {
+	case types.TxTypeAccessList:
+		al := tx.AccessList()
+		result.Accesses = &al
+		result.ChainID = (*hexutil.Big)(tx.ChainId())
+	case types.TxTypeDynamicFee:
+		al := tx.AccessList()
+		result.Accesses = &al
+		result.ChainID = (*hexutil.Big)(tx.ChainId())
+		result.GasFeeCap = (*hexutil.Big)(tx.GasFeeCap())
+		result.GasTipCap = (*hexutil.Big)(tx.GasTipCap())
```

Comment:
Should we calculate gasPrice here?
Or maybe just adding TODO comment also fine for when we change gas policy

Reply:
@jimni1222 
Thanks. I'll add the TODO-tag with description about it.